test/unit/backend_test.rb

```diff
@@ -0,0 +1,100 @@
+# encoding: utf-8
```

Comment:
FYI, We don't need to add these top line utf-8 comments anymore. This changed with 2.0, all 'scripts' are considered to be utf-8 by default.

https://ruby-doc.org/core-2.1.4/Encoding.html#class-Encoding-label-Script+encoding

Reply:
Ah! Awesome. Will fix.